resources/dicts/events/death/general.json

```diff
@@ -1152,387 +1109,475 @@
-            "age": [],
-            "status": [],
+            "age": [ "adolescent", "young adult", "adult", "senior adult" ],
+            "status": ["apprentice", "warrior", "deputy", "leader"],
             "dies": true
         },
         "history": {
+            "cats": "m_c",
             "reg_death": "m_c drowned in a river attempting to escape an enemy attack.",
             "lead_death": "drowned in a river attempting to escape an enemy attack."
         }
     },
     {
-        "event_id": "gen_death_",
+        "event_id": "gen_death_war_any7",
+        "biome": ["any"],
         "camp": [
             "any"
         ],
         "season": [
             "any"
         ],
-        "tags": [
-            "war"
-        ],
+        "sub_type": "war",
+        "tags": [ "some_lives" ],
         "weight": 20,
         "event_text": "m_c fell into a pit trap set by o_c's warriors.",
         "m_c": {
-            "age": [],
-            "status": [],
+            "age": [ "adolescent", "young adult", "adult", "senior adult" ],
+            "status": [ "any" ],
             "dies": true
         },
         "history": {
+            "cats": "m_c",
             "reg_death": "m_c fell into a pit trap set by an enemy Clan.",
             "lead_death": "fell into a pit trap set by an enemy Clan."
         }
     },
     {
-        "event_id": "gen_death_",
+        "event_id": "gen_death_war_any8",
+        "biome": ["any"],
         "camp": [
             "any"
         ],
         "season": [
             "any"
         ],
-        "tags": [
-            "war"
-        ],
+        "sub_type": "war",
+        "tags": [ "some_lives" ],
         "weight": 20,
         "event_text": "m_c was hit by a monster while escaping across a Thunderpath from o_c warriors.",
         "m_c": {
-            "age": [],
-            "status": [],
+            "age": [ "adolescent", "young adult", "adult", "senior adult" ],
+            "status": [ "apprentice", "warrior", "deputy", "leader" ],
             "dies": true
         },
         "history": {
+            "cats": "m_c",
             "reg_death": "m_c was hit by a monster.",
             "lead_death": "{VERB/m_c/were/was} hit by a monster."
         }
     },
     {
-        "event_id": "gen_death_",
+        "event_id": "gen_death_age_any1",
+        "biome": ["any"],
         "camp": [
             "any"
         ],
         "season": [
             "any"
         ],
-        "tags": [
-            "old_age"
-        ],
+        "sub_type": [ "old_age" ],
+        "tags": [ ],
         "weight": 20,
-        "event_text": "m_c wandered out into the territory, and was later found dead, a peaceful expression on {PRONOUN/m_c/poss} face.",
+        "event_text": "m_c wandered out into the territory and was later found dead, a peaceful expression on {PRONOUN/m_c/poss} face.",
         "m_c": {
-            "age": [],
-            "status": [],
+            "age": [ "senior adult", "senior" ],
```

Comment:
you can remove the "senior adult" from here, cats can only die of old_age when they reach a certain age anyways.  the only cats who can get this event are at least 150 moons old.